Makefile

```diff
@@ -121,6 +120,9 @@ test.phpstan: ## Run PHPStan
 test.phpmd: ## Run PHPMD
 	${COMPOSER} phpmd
 
+test.phpspec: ## Run PHPSpec
+	${COMPOSER} phpspec
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** shouldn't it be added to the maker plugin?‏

Reply:
Maybe :D